plugins/database/src/db_plugin.cpp

```diff
@@ -410,14 +419,14 @@ int get_object_count_of_resource_by_name(
             _count = 0;
             return 0;
         }
-        irods::log(PASS(ret));
+        log_db_helper(PASS(ret));
         return ret.code();
     }
 
     std::string resc_id_str;
     ret = irods::lexical_cast<std::string>(resc_id, resc_id_str);
     if(!ret.ok()) {
-        irods::log(PASS(ret));
+        log_db_helper(PASS(ret));
```

Comment:
This can just be `log_db::error` since it's in an error branch.

Reply:
Done.